inspirehep/modules/theme/templates/inspirehep_theme/conferences_in_series.html

```diff
@@ -0,0 +1,27 @@
+{# 
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+#}
+
+<div id="other-conferences-in-the-series">
+  <a href="/conferences/{{record.control_number}}"> {{ record.title }}</a>
+</div>
```

Comment:
These templates are simple, so they could also be done using mustache, which would spare us the weird thing of rendering stuff server-side and then sending the HTML back to the client... what is the reasoning for doing it like that?